odoo/tools/xml_utils.py

```diff
@@ -55,6 +57,37 @@ def resolve(self, url, id, context):
             return self.resolve_string(attachment.raw, context)
 
 
+def test_xsd(url=None, path=None, skip=False):
+    def decorator(func):
+        def wrapped_f(self, *_args):
+            if not skip:
+                xmls = func(self)
```

Comment:
I have move it into `common.py` but not near `_assertXMLEqual`, rather I placed it next to `standalone` first of because it is not meant to be a class method, and because standalone is also a decorator for tests. :smile: